op-chain-ops/upgrades/l1.go

```diff
@@ -168,7 +169,26 @@ func L1ERC721Bridge(batch *safe.Batch, implementations superchain.Implementation
 		return err
 	}
 
-	calldata, err := l1ERC721BridgeABI.Pack("initialize", superchainConfigProxy)
+	var messenger, otherBridge common.Address
+	if config != nil {
```

Comment:
Confused about this check as `config` isn't used in the branch, apologies if it was previously implemented this way

Reply:
We should pull the remote values and compare them against the config if the config exists and error if they are different